app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -296,23 +379,31 @@ const getTablePattern = (index: number) =>
       ),
     ),
   )
+
 function createNode(params: CellClickedEvent) {
-  if (config.nodeType === VECTOR_NODE_TYPE || config.nodeType === COLUMN_NODE_TYPE) {
+  if (config.nodeType === TABLE_NODE_TYPE || config.nodeType === DB_TABLE_NODE_TYPE) {
     config.createNodes({
-      content: getPattern(params.data[INDEX_FIELD_NAME]),
+      content: getTablePattern(params.data[INDEX_FIELD_NAME]),
       commit: true,
     })
   }
-  if (config.nodeType === TABLE_NODE_TYPE) {
+  if (newNodeSelector.value !== undefined && newNodeAction.value) {
     config.createNodes({
-      content: getTablePattern(params.data[INDEX_FIELD_NAME]),
+      content: getAstPattern(params.data[newNodeSelector.value], newNodeAction.value),
       commit: true,
     })
   }
 }
 
-function indexField(): ColDef {
-  return { field: INDEX_FIELD_NAME, onCellClicked: (params) => createNode(params) }
+function toLinkField(fieldName: string): ColDef {
+  return {
+    field: fieldName,
+    onCellDoubleClicked: (params) => createNode(params),
+    tooltipValueGetter: () => {
+      return `Double click to view this ${onClickTooltipValue.value} in seperate node`
```

Comment:
```suggestion
      return `Double click to view this ${onClickTooltipValue.value} in a seperate node`
```

nitpick: I'm not 100% sure but shouldn't there be an article here?